packages/typography/src/styled/StyledEllipsis.ts

```diff
@@ -18,6 +18,7 @@ export const StyledEllipsis = styled.div.attrs({
   text-overflow: ellipsis;
   white-space: nowrap;
   direction: ${props => (props.theme.rtl ? 'rtl' : 'ltr')};
+  color: ${props => getColor({ theme: props.theme, variable: 'foreground.default' })};
```

Comment:
Sets default color correctly based on mode.

Reply:
I don't think we want this; the `color` should be inherited.